server/src/controllers/Events.js

```diff
@@ -2,77 +2,102 @@ const express = require('express');
 const path = require('path');
 const cloudinary = require('cloudinary');
 cloudinary.config({
-	cloud_name: 'coc-vjti',
-	api_key: '552242973352355',
-	api_secret: process.env.CLOUDINARY_SECRET
+  cloud_name: 'coc-vjti',
+  api_key: '552242973352355',
+  api_secret: process.env.CLOUDINARY_SECRET
```

Comment:
Please refrain from uploading api keys publicly. Use them as an environment variable instead.

Reply:
This was already present in the file I didn't add them.

Reply:
Oh well, never noticed this one.